omod/src/main/resources/webModuleApplicationContext.xml

```diff
@@ -129,11 +139,39 @@
 				
 				<prop key="admin/provider/providerAttributeType.list">providerAttributeTypeList</prop>
 				<prop key="admin/provider/providerAttributeType.form">providerAttributeTypeForm</prop>	
-				
+
  			</props>
 		</property>
 	</bean>	
 
+ 	<!-- index pages - begin -->  
+	
+	<!-- admin -->
+	<mvc:view-controller view-name="module/legacyui/admin/index" path="admin/index.htm"/>				
```

Comment:
I'm actually already aware of this, the ideal solution was to map these urls to the Dispatcher servlet from the module, but even the servlet API provides no way to remove a servlet in case the module is stopped/removed. In interest of time, under dev notes on the ticket the only quick solution to find usage of these URLs and change them e.g /openmrs/admin becomes /openmrs/admin/index.jsp